pkg/cluster/px/px.go

```diff
@@ -125,6 +125,7 @@ const (
 	bootstrapCloudDriveNamespace    = "kube-system"
 	internalEtcdConfigMapPrefix     = "px-bootstrap-"
 	cloudDriveConfigMapPrefix       = "px-cloud-drive-"
+	pureCloudDriveConfigMapPrefix   = "px-pure-cloud-drive"
```

Comment:
[nit] it's not a prefix, it's the full name

Reply:
good catch :) Initially I assumed it would be a prefix and forgot to rename after you clarified on slack. Will update.